src/leader_election.rs

```diff
@@ -223,6 +238,7 @@ pub mod ballot_leader_election {
                     self.majority_connected = false;
                 }
 
+                info!(self.logger, "New Leader elected, pid: {}", top_pid);
```

Comment:
```suggestion
                debug!(self.logger, "New Leader elected, pid: {}", top_pid);
```

log leader's ballot as well.